src/funding/paypal/template.jsx

```diff
@@ -234,8 +234,87 @@ export function WalletLabelOld(opts : WalletLabelOptions) : ?ChildType {
     );
 }
 
+function ShowPayLabel(opts) : ?ChildType {
+    const {instrument, content, payNow, textColor, logo, label } = opts;
+
+    return (
+        <div class='show-pay-label'>
+            <div class='pay-label' optional={ 2 }>
+                <Space />
+                {
+                    (instrument && content)
+                        ? <Text>{ payNow ? content.payNow : content.payWith }</Text>
+                        : <Text><PlaceHolder chars={ 7 } color={ textColor } /></Text>
+                }
+                <Space />
+            </div>
+            <div class='logo' optional={ 1 }>
+                {
+                    (instrument && logo)
+                        ? logo
+                        : <Text><PlaceHolder chars={ 4 } color={ textColor } /></Text>
+                }
+            </div>
+            <div class='label'>
+                <Space />
+                {
+                    (instrument && label)
+                        ? <Text>{ label }</Text>
+                        : <Text><PlaceHolder chars={ 6 } color={ textColor } /></Text>
+                }
+            </div>
+        </div>
+    );
+}
+
+function NoPayLabel(opts) : ?ChildType {
```

Comment:
Thoughts on naming this `showBalance()`? I worry that `NoPayLabel` only makes sense after you know there's a `showPayLabel()` function.

Reply:
I see there is not always a balance. This is a tricky parent function to name. Maybe something like `showInstrumentLabel()`

Reply:
Or `showInstrumentOnFile()` 😄

Reply:
I like where you're heading! The component can show more than one instrument, so `showStuffOnFile()`? 🤣 I think `showInstrumentOnFile()` or `showInstrumentsOnFile()` work well.

Reply:
I like `showInstrumentsOnFile()`. Great suggestion 💯